src/serial.rs

```diff
@@ -564,8 +563,13 @@ macro_rules! serialdma {
             }
 
             impl $rxdma {
+                #[deprecated(since = "0.7.1", note = "Please use release instead")]
                 pub fn split(mut self) -> (Rx<$USARTX>, $dmarxch) {
```

Comment:
Could you remove the `mut` here since it gives a warning now that self is just passed along?